### ISSUE TITLE
TCA9548::disableChannel(uint8_t channel) is not working as expected

### ISSUE THREAD

**Issue by pghj** [RobTillaart/TCA9548#20]
I think disableChannel was copy-pasted from enableChannel, but the condition !isEnabled(channel) was not inverted by mistake.

**Comment by RobTillaart** [comment]
Thank you for catching, 
definitely a (copy paste) BUG.
Will fix asap, probably tomorrow.

**Comment by pghj** [comment]
Thanks for the quick response.
It seems to me that the check can just be removed from both methods, because setChannelMask already tests the entire mask for changes.

**Comment by pghj** [comment]
Oops wrong button.

**Comment by RobTillaart** [comment]
> It seems to me that the check can just be removed from both methods, because setChannelMask already tests the entire mask for changes.

Good idea, would simplify code a bit. 
Will create a develop branch asap to prepare for a 0.3.0 version.

**Comment by RobTillaart** [comment]
Propose code 
would also return false if the writing to the multiplexer failed.

```cpp
bool TCA9548::enableChannel(uint8_t channel)
{
  if (channel >= _channels) return false;
  return setChannelMask(_mask | (0x01 << channel));
}

bool TCA9548::disableChannel(uint8_t channel)
{
  if (channel >= _channels) return false;
  return setChannelMask(_mask & ~(0x01 << channel));
}
```

**Comment by RobTillaart** [comment]
@pghj 
Develop branch created, build-CI is running.
Some more edits were made.

**Comment by pghj** [comment]
> Propose code

Looks good. Can you also make it return the result of setChannelMask when calling selectChannel?

**Comment by RobTillaart** [comment]
Will check after lunch

**Comment by RobTillaart** [comment]
> > Propose code
> 
> Looks good. Can you also make it return the result of setChannelMask when calling selectChannel?

done

**Comment by RobTillaart** [comment]
@pghj 
Do you have time to test the develop branch, otherwise I will merge it (as I see no issues for now).

**Comment by RobTillaart** [comment]
@pghj 
I merged develop and released 0.3.0 .

Again thanks for reporting the issue!
